cytotable/presets.py

```diff
@@ -317,6 +317,33 @@
         # and modified at runtime as needed
         "CONFIG_JOINS": "",
     },
+    "deepprofiler": {
+        # version specifications using related references
+        "CONFIG_SOURCE_VERSION": {
+            "deepprofiler": "v0.3.1",
+            "cellprofiler": "v4.2.x",
+        },
+        # names of source table compartments (for ex. cells.csv, etc.)
+        "CONFIG_NAMES_COMPARTMENTS": tuple(),
+        # names of source table metadata (for ex. image.csv, etc.)
+        "CONFIG_NAMES_METADATA": tuple(),
+        # column names in any compartment or metadata tables which contain
+        # unique names to avoid renaming
+        "CONFIG_IDENTIFYING_COLUMNS": tuple(),
+        # pagination keys for use with this data
+        # of the rough format "table" -> "column".
+        # note: page keys are expected to be numeric (int, float)
+        "CONFIG_PAGE_KEYS": {
+            "join": "Metadata_Site",
```

Comment:
hmm, for single cells, isn't there an additional identifier?

Reply:
Where we could normally use object numbers I didn't find this within the data and so felt this might be the next best thing. This might be a gap in my understanding (I couldn't find a distinct way to relate the object numbers back to the table, mostly because they didn't exist within the NPZ from what I saw).

Reply:
In the case of the test data, I believe we're constricted to this index file's columns (which then go on to be included within the NPZ metadata): https://github.com/cytomining/pycytominer/blob/main/tests/test_data/DeepProfiler_example_data/inputs/metadata/test_index.csv .

Reply:
Thanks @gwaybio ! After you mentioned this I realized I hadn't fully matched how location and feature data are treated when exported from NPZ to Parquet through this PR (these were single columns with array values). I've now shifted this to match how Pycytominer treats them, as new columns per array value (and changed the tests associated with this).